_aggregations/metric/matrix-stats.md

```diff
@@ -9,8 +9,23 @@ redirect_from:
 
 # Matrix stats aggregations
 
-The `matrix_stats` aggregation generates advanced stats for multiple fields in a matrix form.
-The following example returns advanced stats in a matrix form for the `taxful_total_price` and `products.base_price` fields:
+The `matrix_stats` metric is a multi-value metric that generates covariance statistics for two or more fields in matrix form. 
```

Comment:
1. Does `matrix-stats` always sample every available document? If not, how is sampling done?

2. What assumptions does `matrix-stats` make about independence of data samples? Is it sufficent to say that it assumes documents are independent of each other?

Reply:
Again will probably keep it as `multi-value metric aggregation` and maybe link to the metric aggregation page

Reply:
@jainankitk Please see my two technical questions above. Can you provide any insight?

Reply:
@jainankitk Could you please answer this quiestion?

Reply:
> Does matrix-stats always sample every available document? If not, how is sampling done?

I don't think there is any sampling, all the documents are processed

> What assumptions does matrix-stats make about independence of data samples? Is it sufficent to say that it assumes documents are independent of each other?

`Matrix-stats` does not make any assumptions about the data. I am wondering why does it need to make any assumption?

Reply:
@jainankitk There are several assumptions that have to be met for a correlation to be valid: https://www.statology.org/pearson-correlation-assumptions/. However, that level of detail is probably beyond the scope of the documentation here. At some level, the user is assumed to know what they're doing if they use an aggregation.